keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -65,7 +65,7 @@
             }
           },
           "default": {
-            "description": "Default response. It will be deserialized as per the error definition specified in the schema. An exception will be thrown.",
+            "description": "CDN error response describing why the operation failed.",
```

Comment:
Why is termed as the "CDN error response"? We have a service named "CDN". However, this error response is from the KeyVault service? Is this a copy paste error ?